docs/release-source/release/spies.md

```diff
@@ -87,7 +87,7 @@ all calls. The following is a slightly contrived example:
 
 Spies provide a rich interface to inspect their usage. The above examples showed
 the `calledOnce` boolean property as well as the `getCall` method and the
-returned object's `args` property. There are three ways of inspecting call data.
+returned object's `args` property. There are three ways of inspecting [call][call] data.
```

Comment:
Smarty pants comment: `[call][call]` == `[call][]`

Reply:
Interesting! (and a little confusing)